askapp/views.py

```diff
@@ -204,22 +205,21 @@ def registration_allowed(self):
 
     @staticmethod
     def is_email_blacklisted(email):
-        return any([email.lower().endswith('@'+d) for d in settings.BLACKLISTED_DOMAINS])
-
-    def get_email_context(self, activation_key):
-        """
-        populate template variables for the activation email
-        """
-        result = super(AskappRegistrationView, self).get_email_context(activation_key)
-        d = {'domain': self.request.get_host(), 'name': config.SITE_NAME}
-        result['site'] = namedtuple("Site", d.keys())(*d.values())
-        return result
-
-    def send_activation_email(self, user):
-        if self.is_email_blacklisted(user.email):
-            logging.debug('blacklisted email %s', user.email)
+        try:
+            sfs_url = f'http://api.stopforumspam.org/api?email={email}'
```

Comment:
we could later add cleaning of emails to remove any part after(and including) "+" in @gmail.com adresses